### ISSUE TITLE
playwright._impl._api_types.TimeOutError: Timeout 30000ms exceeded.

### ISSUE THREAD

**Issue by 0x5345414E** [0x5345414E/AutoDev#4]
playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
waiting for locator("#ct100_MainContent_ddlClientStatus_Arrow")

in ```unknownPhys```

**Comment by 0x5345414E** [comment]
adding global timeout

**Comment by 0x5345414E** [comment]
Global timeout didn't work. Need to find new selector as this fails on both ```unkwn``` and in ```non_admits``` 

Considering locator as x-path. Checking this today.

**Comment by 0x5345414E** [comment]
```
    def unkwn(page):
    logging.info('Pulling Unknown Physicians report...')
    page.goto("https://hh.kantimehealth.net/Z1/UI/Billing/Reports/Report_ClientMaster.aspx?Reference=1&HelpContextNo=3&MenuItemID=352")
    if page.url != "https://hh.kantimehealth.net/Z1/UI/Billing/Reports/Report_ClientMaster.aspx?Reference=1&HelpContextNo=3&MenuItemID=352":
        print("URL does not match. Session expired or URL is bad.")
        auth(page)
    else:
        print("Pulling Unknown Physicians report...")
    page.locator("#ctl00_MainContent_ddlClientStatus_Arrow").click()
    page.get_by_label("Transferred").uncheck()
    page.get_by_label("On Hold").uncheck()
    page.get_by_label("Discharged").uncheck()
    page.get_by_label("Pending SOC").uncheck()
    page.get_by_role("cell", name="Episode End From: To:").click()
    page.get_by_title("Please click here to set more filter option(s).").click()
    page.get_by_role("row", name="Attending Physician:").get_by_role("textbox").click()
    page.get_by_role("row", name="Attending Physician:").get_by_role("textbox").fill("unkn")
    page.get_by_role("row", name="Attending Physician:").get_by_role("textbox").press("Enter")
    page.locator("li").filter(has_text="UNKNOWN, PHYSICIAN DO NOT USE").click()
    page.get_by_role("button", name="Done").click()
    page.get_by_role("button", name="Display").click()
    with page.expect_download(timeout=0) as download_info: #
        page.get_by_role("img", name="Export To Excel", exact=True).click
    ext = ".csv"
    my_downloader = MyDownloader(ext) # Provide the 'ext' argument here
    download = download_info.value # Retain Download Value.
    path = my_downloader.file_path(name="Unknown_Physician") # Note that the 'file' argument has been changed to 'name'
    download.save_as(path)
```

The issue for the timeout happens on line 143 in `reportRun` on both main and staging branch. Not sure if I can set a manual timeout but the automation stops after this. Haven't attempted to work with x-paths yet, and I have been considering rewriting this whole project in Selenium already.

**Comment by 0x5345414E** [comment]
resolved with set page.timeout = 0 and swapped to xpath to fix this and utilizing xpath locators. for non_admits.

**Comment by 0x5345414E** [comment]
This has been fixed. Adding a timeout/explicit wait feature to allow time for browser context to load dropdown from javascript function. Tested and confirmed working. Latest commit fixed this and issue #3
